components/about/Resume.tsx

```diff
@@ -72,7 +72,7 @@ const Resume: React.FC<ResumePageProps> = (props) => {
         Take a look at my{" "}
         <span className="leading-ti bg-portfolio-main px-1">resume</span>
       </h2>
-      <ul className="mx-auto mt-8 flex w-full flex-col items-center justify-center gap-8">
+      <ul className="mx-auto mt-8 mb-8 flex w-full flex-col items-center justify-center gap-8">
```

Comment:
Hey!! Thanks for pointing this out, you have a sharp eye 🕵️ .
In most pages I've maintained a pattern of keeping the margin-bottom twice the size of the margin-top.
Can you update this same here from `mb-8` to `mb-16`?

Reply:
Sure, give me a few mins and I will update that

Reply:
Awesome, thanks, buddy.